MM/6.3.0/asset_relation_types/id_360_images.dcl

```diff
@@ -0,0 +1,53 @@
+resource asset_relation_type id_360_images {
+    name = '360 Images'
+    description = 'The primary asset serves as a placeholder for displaying secondary assets within a 360° viewer.'
+    multiplicity = 'OneToMany'
+    is_locked = true
+    show_in_list_when_primary = true
+    show_in_list_when_secondary = true
+    additional_fields_when_primary = []
+    additional_fields_when_secondary = []
+    primary_asset_categories = []
+    secondary_asset_categories = []
+    channel_publishing_behavior = {
+        enable_behavior = true
+        block_publishing = false
+        inherit_channel_folders = true
+    }
+    item_security_behavior = {
+        enable_behavior = true
+        inheritance = 'InheritRead'
+    }
+    deletion_behavior = {
+        enable_behavior = true
+        inherit_soft_delete = true
+        inherit_hard_delete = true
```

Comment:
Considering we are inheriting delete, don't we want to InheritReadWrite instead?